src/components/VHeader/VWordPressLink.vue

```diff
@@ -10,8 +13,7 @@
       class="label-regular flex flex-row items-center gap-1"
     >
       <template #wordpress>
-        <WordPress class="aria-hidden" />
```

Comment:
`class="aria-hidden"`? I think just making this bit `aria-hidden=true` would've fixed it.

Reply:
Do you mean removing aria-label added in this PR and adding the non-tailwind aria-hidden class would fix the "no space between WordPress and project for screenreaders" problem, @dhruvkb?
I tried with VoiceOver, and when you and aria-label to the link, all of the other content becomes invisible for screenreader.

Reply:
Huh, that's interesting. I approved your solution though because it definitely works.